core/src/main/scala/org/broadinstitute/dsde/rawls/billing/GoogleBillingProjectLifecycle.scala

```diff
@@ -37,6 +34,8 @@ class GoogleBillingProjectLifecycle(
 ) extends BillingProjectLifecycle {
   implicit val errorReportSource: ErrorReportSource = ErrorReportSource("rawls")
 
+  override val deleteJobType: JobType = GoogleBillingProjectDelete
```

Comment:
making this part of the `BillingProjectLifecycle` makes the orchestrator more open-ended and generic

Reply:
Is the variable actually needed though? I don't see that anything besides a test references it.

Reply:
The orchestrator uses it to determine the type of job it needs to schedule, instead of hard-coding the logic there.
This makes it so all of the specific logic, outside of determining which lifecycle to use, is all contained here.

Reply:
Right, I see it now.